src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -468,19 +468,27 @@ export class SavedObjectsClient {
    * Remove a saved object from workspaces
    * @param type
    * @param id
-   * @param workspaces
+   * @param options
    */
-  deleteFromWorkspaces = async <T = unknown>(type: string, id: string, workspaces: string[]) => {
+  deleteFromWorkspaces = async <T = unknown>(
+    type: string,
+    id: string,
+    options: { workspaces?: string[] }
```

Comment:
Then I think the type of `options` should be extending `SavedObjectsBaseOptions`?

Reply:
Updated.